inst/ms/aggreCAT.qmd

```diff
@@ -961,7 +1039,7 @@ to
 `r focal_claims %>% dplyr::filter(paper_id %in% c(108, 138)) %>% BayesianWAgg(type = "BayTriVar", percent_toggle = TRUE) %>% dplyr::filter(paper_id == 108) %>% pluck("cs") %>% round(2)`.
 Note that [BayesianWAgg]{.fct} cannot calculate confidence scores when
```

Comment:
Should also note that while it will work for n=2, it's up to the user to determine their own justifiable min(n) for reliable/appropriate variance calculations